WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -51,12 +51,20 @@ open class WordPressOrgXMLRPCApi: NSObject {
         let sessionManager = Alamofire.SessionManager(configuration: sessionConfiguration)
 
         let sessionDidReceiveChallengeWithCompletion: ((URLSession, URLAuthenticationChallenge, @escaping(URLSession.AuthChallengeDisposition, URLCredential?) -> Void) -> Void) = { [weak self] session, authenticationChallenge, completionHandler in
-            return self?.urlSession(session, didReceive: authenticationChallenge, completionHandler: completionHandler)
+            guard let strongSelf = self else {
+                return
+            }
+
+            return strongSelf.urlSession(session, didReceive: authenticationChallenge, completionHandler: completionHandler)
```

Comment:
@jkmassel I think it's sufficient to not return the result here. The return type is `Void` anyway.

This means we don't need the `guard let strongSelf = self` above either.

Reply:
Yep you're right!

Thanks @bjtitus!!